Solver_interface/include/CGAL/Eigen_diagonalize_traits.h

```diff
@@ -150,11 +150,11 @@ class Eigen_diagonalize_traits
 
     if(res)
     {
-      for(std::size_t i=0; i<dim; ++i)
+      for(int i=0; i<dim; ++i)
       {
         eigenvalues[i] = static_cast<FT>(eigenvalues_[i]);
 
-        for(std::size_t j=0; j<dim; ++j)
+        for(int j=0; j<dim; ++j)
```

Comment:
I do not understand why that was committed in this PR. And the patch is wrong, because it triggered a lot of warnings in Classification, Jet_fitting, PSP, Ridges, Scale_space... 

https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-25.shtml

Reply:
I will revert the patch, and test the same PR again tonight.